ui/src/mixins/MobileLayout.js

```diff
@@ -0,0 +1,209 @@
+export default {
+    methods: {
+        gridLayoutAddRecord(screenSize){
```

Comment:
This method name is confusing. It implies that you are adding a record which is not the case.